### PR TITLE
Updated referral codes.

### PR DESCRIPTION
**Pull request by mvastarelli**
Fixed a couple of referral codes.

@evan-cohen Please check the notes below before signing off on this.

### CODE REVIEW

File: src/models/CoinDefinition.ts [hunk 1]
```diff
@@ -192,7 +192,7 @@ export const ALL_COINS: CoinDefinition[] = [
     name: 'Dogecoin',
     symbol: 'DOGE',
     blockchains: ['DOGE'],
-    referrals: ['pdwb-af48', '1f6v-fy9m'],
+    referrals: ['3q5x-94gx', '1f6v-fy9m'],
```
Comment (mvastarelli) [inline src/models/CoinDefinition.ts:195]: @evan-cohen please confirm this isn't your referral code that I wiped out.

File: src/models/CoinDefinition.ts [hunk 2]
```diff
@@ -352,7 +352,7 @@ export const ALL_COINS: CoinDefinition[] = [
     name: 'PancakeSwap',
     symbol: 'CAKE',
     blockchains: ['BSC'],
-    referrals: ['i0kx-z4kc'],
+    referrals: ['ofq5-w659'],
```
Comment (mvastarelli) [inline src/models/CoinDefinition.ts:355]: @evan-cohen Can you verify that this code isn't yours either.

File: src/models/CoinDefinition.ts (hunk 1 above)
Comment (evan-cohen) [reply src/models/CoinDefinition.ts:195]: all set - not mine.

File: src/models/CoinDefinition.ts (hunk 2 above)
Comment (evan-cohen) [reply src/models/CoinDefinition.ts:355]: not mine

**Review by evan-cohen — APPROVED** [review]

**Pull request merged**